torch/quantization/observer.py

```diff
@@ -139,6 +139,23 @@ def calculate_qparams(self):
     def extra_repr(self):
         return 'min_val={}, max_val={}'.format(self.min_val, self.max_val)
 
+
+class DebugObserver(ObserverBase):
```

Comment:
Maybe rename it to TensorObserver to be more general since it's mainly observing tensor values, which could be used for other purposes than debugging :-)